### ISSUE TITLE
"vizro" is registered but the path requested is not valid.

### ISSUE THREAD

**Issue by KobaKhit** [mckinsey/vizro#836]
### Which package?

vizro

### Package version

0.1.25

### Description

I am on a windows 10 machine. Followed the installation instructions from the official docs. When running the example app, dash cannot load the static assets.

![image](https://github.com/user-attachments/assets/94559e37-005b-495b-9da8-37a9c4e6d32f)

**Steps**
  - 1. `conda create --name vizro-environment`
        `conda activate vizro-environment`

  - 2. `pip install vizro`

  - 3. run the app `python app.py`

**app.py**
```python
"""Dev app to try things out."""

import pandas as pd
import vizro.models as vm
import vizro.plotly.express as px
from vizro import Vizro
from vizro._themes._color_values import COLORS

pastry = pd.DataFrame(
    {
        "pastry": [
            "Scones",
            "Bagels",
            "Muffins",
            "Cakes",
            "Donuts",
            "Cookies",
            "Croissants",
            "Eclairs",
            "Brownies",
            "Tarts",
            "Macarons",
            "Pies",
        ],
        "Profit Ratio": [-0.10, -0.15, -0.05, 0.10, 0.05, 0.20, 0.15, -0.08, 0.08, -0.12, 0.02, -0.07],
    }
)


page = vm.Page(
    title="Test New ZZZZZZZ",
    components=[
        vm.Graph(
            figure=px.bar(
                pastry.sort_values("Profit Ratio"),
                orientation="h",
                x="Profit Ratio",
                y="pastry",
                color="Profit Ratio",
                color_continuous_scale=COLORS["DIVERGING_RED_CYAN"],
            ),
        ),
    ],
)

dashboard = vm.Dashboard(pages=[page],theme="vizro_light")

if __name__ == "__main__":
    Vizro().build(dashboard).run()
```

**Traceback after loading the page**
```
Desktop\game intelligence platform> python app.py
Dash is running on http://127.0.0.1:8050/

INFO:dash.dash:Dash is running on http://127.0.0.1:8050/

 * Serving Flask app 'app'
 * Debug mode: off
INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:8050
INFO:werkzeug:Press CTRL+C to quit
INFO:werkzeug:127.0.0.1 - - [28/Oct/2024 17:13:51] "GET / HTTP/1.1" 200 -
ERROR:app:Exception on /_dash-component-suites/vizro/static/css/vizro-bootstrap.v0_1_25m1730149998.min.css [GET]
Traceback (most recent call last):
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\dash\dash.py", line 995, in serve_component_suites
    _validate.validate_js_path(self.registered_paths, package_name, path_in_pkg)
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\dash\_validate.py", line 382, in validate_js_path
    raise exceptions.DependencyException(
dash.exceptions.DependencyException: "vizro" is registered but the path requested is not valid.
The path requested: "static/css/vizro-bootstrap.min.css"
List of registered paths: defaultdict(<class 'set'>, {'dash': {'dcc/async-markdown.js.map', 'dcc/async-datepicker.js', 'dcc/async-dropdown.js.map', 'dcc/dash_core_components.js', 'dcc/async-mathjax.js.map', 'dcc/async-upload.js', 'dash_table/async-export.js.map', 'dcc/async-datepicker.js.map', 'dcc/async-graph.js', 'dcc/async-highlight.js', 'dcc/async-highlight.js.map', 'dcc/async-dropdown.js', 'dcc/dash_core_components-shared.js.map', 'deps/prop-types@15.8.1.min.js', 'dash_table/bundle.js.map', 'dash-renderer/build/dash_renderer.min.js.map', 'deps/react@16.14.0.min.js', 'html/dash_html_components.min.js', 'dash_table/async-table.js', 'deps/react-dom@16.14.0.min.js', 'dash_table/async-highlight.js.map', 'dcc/dash_core_components.js.map', 'html/dash_html_components.min.js.map', 'dcc/async-graph.js.map', 'dcc/async-slider.js', 'dcc/async-slider.js.map', 'dash-renderer/build/dash_renderer.min.js', 'dash_table/bundle.js', 'dcc/dash_core_components-shared.js', 'dash_table/async-export.js', 'dash_table/async-highlight.js', 'dcc/async-upload.js.map', 'deps/polyfill@7.12.1.min.js', 'dcc/async-markdown.js', 'dcc/async-mathjax.js', 'dash_table/async-table.js.map'}, 'dash_bootstrap_components': {'_components/dash_bootstrap_components.min.js'}, 'dash_mantine_components': {'dash_mantine_components.js.map', 'dash_mantine_components.js'}, 'vizro': {'static\\js\\models\\slider.js', 'static\\css\\fonts\\material-symbols-outlined.woff2', 'static\\css\\index.css', 'static\\css\\code.css', 'static\\css\\fonts\\inter-variable-font.ttf', 'static\\images\\error_404.svg', 'static\\css\\variables.css', 'static\\js\\actions\\build_action_loop_callbacks.js', 'static\\css\\token_names.css', 'static\\css\\loading.css', 'static\\css\\tabs.css', 'static\\css\\images.css', 'static\\css\\tooltip.css', 'static\\js\\models\\date_picker.js', 'static\\css\\collapse.css', 'static\\css\\vizro-bootstrap.min.css', 'static\\css\\tiles.css', 'static\\css\\figures.css', 'static\\css\\aggrid.css', 'static\\css\\datepicker.css', 'static\\css\\dropdown.css', 'static\\css\\table.css', 'static\\js\\models\\dashboard.js', 'static\\css\\bootstrap_overwrites.css', 'static\\css\\scroll_bar.css', 'static\\js\\models\\range_slider.js', 'static\\css\\toggle.css', 'static\\css\\layout.css', 'static\\css\\slider.css', 'static\\css\\container.css'}, 'plotly': {'package_data/plotly.min.js'}})
INFO:werkzeug:127.0.0.1 - - [28/Oct/2024 17:13:51] "GET /_dash-component-suites/vizro/static/css/vizro-bootstrap.v0_1_25m1730149998.min.css HTTP/1.1" 500 -

```

### How to Reproduce

  - 1. `conda create --name vizro-environment`
        `conda activate vizro-environment`

  - 2. `pip install vizro`

  - 3. run the app `python app.py`

### Output

```
Desktop\game intelligence platform> python app.py
Dash is running on http://127.0.0.1:8050/

INFO:dash.dash:Dash is running on http://127.0.0.1:8050/

 * Serving Flask app 'app'
 * Debug mode: off
INFO:werkzeug:WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:8050
INFO:werkzeug:Press CTRL+C to quit
INFO:werkzeug:127.0.0.1 - - [28/Oct/2024 17:13:51] "GET / HTTP/1.1" 200 -
ERROR:app:Exception on /_dash-component-suites/vizro/static/css/vizro-bootstrap.v0_1_25m1730149998.min.css [GET]
Traceback (most recent call last):
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 1473, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 882, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 880, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\flask\app.py", line 865, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\dash\dash.py", line 995, in serve_component_suites
    _validate.validate_js_path(self.registered_paths, package_name, path_in_pkg)
  File "C:\Users\kkhitalishvili\AppData\Local\miniconda3\lib\site-packages\dash\_validate.py", line 382, in validate_js_path
    raise exceptions.DependencyException(
dash.exceptions.DependencyException: "vizro" is registered but the path requested is not valid.
The path requested: "static/css/vizro-bootstrap.min.css"
List of registered paths: defaultdict(<class 'set'>, {'dash': {'dcc/async-markdown.js.map', 'dcc/async-datepicker.js', 'dcc/async-dropdown.js.map', 'dcc/dash_core_components.js', 'dcc/async-mathjax.js.map', 'dcc/async-upload.js', 'dash_table/async-export.js.map', 'dcc/async-datepicker.js.map', 'dcc/async-graph.js', 'dcc/async-highlight.js', 'dcc/async-highlight.js.map', 'dcc/async-dropdown.js', 'dcc/dash_core_components-shared.js.map', 'deps/prop-types@15.8.1.min.js', 'dash_table/bundle.js.map', 'dash-renderer/build/dash_renderer.min.js.map', 'deps/react@16.14.0.min.js', 'html/dash_html_components.min.js', 'dash_table/async-table.js', 'deps/react-dom@16.14.0.min.js', 'dash_table/async-highlight.js.map', 'dcc/dash_core_components.js.map', 'html/dash_html_components.min.js.map', 'dcc/async-graph.js.map', 'dcc/async-slider.js', 'dcc/async-slider.js.map', 'dash-renderer/build/dash_renderer.min.js', 'dash_table/bundle.js', 'dcc/dash_core_components-shared.js', 'dash_table/async-export.js', 'dash_table/async-highlight.js', 'dcc/async-upload.js.map', 'deps/polyfill@7.12.1.min.js', 'dcc/async-markdown.js', 'dcc/async-mathjax.js', 'dash_table/async-table.js.map'}, 'dash_bootstrap_components': {'_components/dash_bootstrap_components.min.js'}, 'dash_mantine_components': {'dash_mantine_components.js.map', 'dash_mantine_components.js'}, 'vizro': {'static\\js\\models\\slider.js', 'static\\css\\fonts\\material-symbols-outlined.woff2', 'static\\css\\index.css', 'static\\css\\code.css', 'static\\css\\fonts\\inter-variable-font.ttf', 'static\\images\\error_404.svg', 'static\\css\\variables.css', 'static\\js\\actions\\build_action_loop_callbacks.js', 'static\\css\\token_names.css', 'static\\css\\loading.css', 'static\\css\\tabs.css', 'static\\css\\images.css', 'static\\css\\tooltip.css', 'static\\js\\models\\date_picker.js', 'static\\css\\collapse.css', 'static\\css\\vizro-bootstrap.min.css', 'static\\css\\tiles.css', 'static\\css\\figures.css', 'static\\css\\aggrid.css', 'static\\css\\datepicker.css', 'static\\css\\dropdown.css', 'static\\css\\table.css', 'static\\js\\models\\dashboard.js', 'static\\css\\bootstrap_overwrites.css', 'static\\css\\scroll_bar.css', 'static\\js\\models\\range_slider.js', 'static\\css\\toggle.css', 'static\\css\\layout.css', 'static\\css\\slider.css', 'static\\css\\container.css'}, 'plotly': {'package_data/plotly.min.js'}})
INFO:werkzeug:127.0.0.1 - - [28/Oct/2024 17:13:51] "GET /_dash-component-suites/vizro/static/css/vizro-bootstrap.v0_1_25m1730149998.min.css HTTP/1.1" 500 -
```

### Code of Conduct

- [X] I agree to follow the [Code of Conduct](https://github.com/mckinsey/vizro/blob/main/CODE_OF_CONDUCT.md).

**Comment by KobaKhit** [comment]
I think I know what the issue is. I should have mentioned that my desktop folder is actually a OneDrive folder. I will try running the app on my local desktop folder and will report here.

**Comment by KobaKhit** [comment]
Nope. The issue still persists even after running the app in the local Desktop folder.

**Comment by Joseph-Perkins** [comment]
Hi @KobaKhit, thank you for the very clear explanation! We will investigate to determine what might be the cause, and will reply here with any updates 👍

**Comment by antonymilne** [comment]
Hi @KobaKhit, thank you indeed for this very useful bug report. I've tried to do a fix, and it would be great if you could test it before we release it.

Please can you run these commands and then try again and see if it works? 🙏 
```
pip uninstall vizro
pip install git+https://github.com/mckinsey/vizro.git@fix/windows-paths#subdirectory=vizro-core
```

**Comment by KobaKhit** [comment]
Yep, now its working without any issues. Thank you!

**Comment by antonymilne** [comment]
Great, thank you for checking! I'll finish off the fix and then release 0.1.26 soon.

**Comment by antonymilne** [comment]
FYI 0.1.26 is now released with the fix. Thank you for your bug report @KobaKhit.
